cloud_node:
  ros__parameters:
    min_distance: 0.15                 
    max_distance: 150.0                
    degree_mode: 2 
    distance_unit: 0.25
    return_mode: 1
    config_vert: true 
    print_vert: false  
    scan_frame_id: laser_link
    scan_num: 15
    publish_scan: true
    
lidar_node:
  ros__parameters:
    device_ip: 192.168.1.200
    msop_port: 2370
    difop_port: 2371
    frame_id: laser_link       
    add_multicast: false
    group_ip: 224.1.1.2
    rpm: 600.0
    return_mode: 1
    degree_mode: 2        
    time_synchronization: false    
    
